api/v1/flow/flow.proto

```diff
@@ -335,15 +335,15 @@ enum DropReason {
     // non-drop reasons
     DROP_REASON_UNKNOWN = 0;
     // drop reasons
-    INVALID_SOURCE_MAC = 130;
-    INVALID_DESTINATION_MAC = 131;
+    INVALID_SOURCE_MAC = 130 [deprecated = true];
```

Comment:
Are these deprecations a consequence of the new drop reason?  If not it might be worth putting this into a separate commit to separate-out & explain the change.

Reply:
I initially lobbed it in to avoid regenerating the proto twice, easing backports, but with the IGMP drop reasons  added in between we'll need to regenerate when backporting anyway.